pkg/fixtures/role.go

```diff
@@ -0,0 +1,14 @@
+package fixtures
+
+import (
+	"github.com/stackrox/rox/generated/storage"
+)
+
+// GetScopedK8SRole returns a mock K8SRole belonging to the input scope.
+func GetScopedK8SRole(id string, clusterID string, namespace string) *storage.K8SRole {
+	return &storage.K8SRole{
+		Id:        id,
+		Namespace: namespace,
+		ClusterId: clusterID,
```

Comment:
I think the postgres code generator relies on some utility to fill in the proto structure field with random data. It might be worth looking if this can be re-used in the fixtures (although for the purpose of SAC tests, ClusterID and Namespace are enough).

Reply:
Yeah, I saw this as well. Sadly, I would have to do bigger refactorings than I'd ideally like. If we import `testutils` within `fixtures`, this will lead to a cyclic import within the `blevesearch` package. So, for now refrained from doing that.